### PR TITLE
Finish job bind feature

### PR DESCRIPTION
**Pull request by crgisch**
Finish missing actions for job bind
Add tests for existent bind actions
Move job to servicemanager

### CODE REVIEW

File: job/job.go [hunk 1]
```diff
@@ -155,8 +163,73 @@ func UpdateJob(ctx context.Context, newJob, oldJob *jobTypes.Job, user *auth.Use
...
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (*jobService) RemoveInstance(ctx context.Context, job *jobTypes.Job, removeArgs jobTypes.RemoveInstanceArgs) error {
+	lenBefore := len(job.Spec.ServiceEnvs)
+	for i := 0; i < len(job.Spec.ServiceEnvs); i++ {
```
Comment (arthurcgc) [inline job/job.go:192]:
```suggestion
i := 0
	for _, serviceEnv := range job.Spec.ServiceEnvs {
		if serviceEnv.ServiceName == removeArgs.ServiceName && serviceEnv.InstanceName == removeArgs.InstanceName {
			job.Spec.ServiceEnvs = append(job.Spec.ServiceEnvs[:i], job.Spec.ServiceEnvs[i+1:]...)
			i--
		}
		i++
	}
```

File: job/job.go [hunk 2]
```diff
@@ -155,8 +163,73 @@ func UpdateJob(ctx context.Context, newJob, oldJob *jobTypes.Job, user *auth.Use
...
+		se := job.Spec.ServiceEnvs[i]
+		if se.ServiceName == removeArgs.ServiceName && se.InstanceName == removeArgs.InstanceName {
+			job.Spec.ServiceEnvs = append(job.Spec.ServiceEnvs[:i], job.Spec.ServiceEnvs[i+1:]...)
+			i--
+		}
+	}
+
+	toUnset := lenBefore - len(job.Spec.ServiceEnvs)
```
Comment (arthurcgc) [inline job/job.go:200]:
```suggestion
	toUnset := lenBefore - i
```

**Review by arthurcgc — CHANGES_REQUESTED** [review]
This is just half of the review, on Monday I'll revisit this

**Review by arthurcgc — CHANGES_REQUESTED** [review]
There are no unsetJobEnv, unbindJobServiceInstance, bindJobServiceInstance unit tests...

**Review by wpjunior — APPROVED** [review]

**Pull request merged**